roles/create-openshift-resources/tasks/create_environment_variables_string.yml

```diff
@@ -1,13 +1,7 @@
 ---
-- name: "Set Base environment_variables_string"
-  set_fact:
-    environment_variables_string: '--env '
 
 - name: "Add variable to environment_variables_string"
   set_fact:
-    environment_variables_string: "{{ environment_variables_string }}{{ item }}={{ environment_variables[item] }},"
+    environment_variables_string: "{{ environment_variables_string }} -e '\"{{ item }}={{ environment_variables[item] }}\"' "
```

Comment:
it is already defined in create_app.yml. https://github.com/rht-labs/ansible-stacks/blob/master/roles/create-openshift-resources/tasks/create_app.yml#L26.

It is validated in the test. There are no env vars in pipeline delivery.